__tests__/app.test.js

```diff
@@ -76,5 +75,65 @@ describe('GET /api/articles/:article_id',()=> {
             expect(response.body.message).toBe('Bad request')
         })
     })
-//ndeeferf
+
+})
+describe('GET /api/articles', () => {
+    it('GET: 200 responds with appropriate status and with an array of articles',() => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) =>{
+            const articles = response.body.articles
+            expect(Array.isArray(articles)).toBe(true)
+            expect(articles.length).toBeGreaterThan(0)
+            articles.forEach((article) =>{
+                expect(article).toHaveProperty('author')
+                expect(article).toHaveProperty('title')
+                expect(article).toHaveProperty('article_id')
+                expect(article).toHaveProperty('topic')
+                expect(article).toHaveProperty('votes')
+                expect(article).toHaveProperty('article_img_url')
+                expect(article).toHaveProperty('comment_count')
+                expect(article).not.toHaveProperty('body')
+            })
+        })
+    })
+    it('should return an array of articles in descending order by default', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.articles.length).toBeGreaterThan(0)
+            expect(response.body.articles).toBeSortedBy('created_at',
+                {descending: true})
+        })
+    })
+    it('should return an array of articles by the given sort by query',() => {
+        return request(app)
+        .get('/api/articles?sort_by=votes')
+        .expect(200)
+        .expect(200)
+        .then((response) => {
```

Comment:
I believe that this functionality will come later. Please only focus on what the ticket asks for :)